shared/components/StudyInfo/LearningPeriod.tsx

```diff
@@ -149,6 +138,24 @@ export class LearningPeriod extends React.Component<LearningPeriodProps> {
             </HMediaQuery.MaxWidth>
           </React.Fragment>
         )}
+
+        {muutOppimisymparistot.map((environment, i) => {
+          const nimi =
```

Comment:
const {nimi = ""} = environment.oppimisymparisto

Reply:
oppimisymparisto for muut environment is mandatory so optional checks could be removed